OpenRA.Mods.Common/Scripting/Properties/CombatProperties.cs

```diff
@@ -73,5 +73,17 @@ public void PatrolUntil(CPos[] waypoints, LuaFunction func, int wait = 0)
 				using (var f = func.CopyReference() as LuaFunction)
 					Self.QueueActivity(new CallFunc(() => PatrolUntil(waypoints, f, wait)));
 		}
+
+		[ScriptActorPropertyActivity]
+		[Desc("Move to a cell and demolish whatever is there.")]
+		public void GoDemolish(Actor target)
```

Comment:
This will need to move into its own `DemolishProperties` that implements `Requires<C4Demolition>` in order for it to behave correctly in all cases.